dsc/getting-started/winGettingStarted.md

```diff
@@ -0,0 +1,116 @@
+---
+ms.date:  08/15/2019
+keywords:  dsc,powershell,configuration,setup
+title:  Get started with Desired State Configuration (DSC) for Windows
+---
```

Comment:
ms.service: automation
author: bobbytreed // So that I get tagged on issues
ms.author: robreed // So that it shows up in other reports
ms.topic: article
manager: carmonm